strictdoc/backend/excel/import_/excel_sheet_proxy.py

```diff
@@ -15,56 +21,61 @@ def __init__(self, file: str):
         if file.endswith(".xlsx"):
             self.workbook = openpyxl.load_workbook(file)
             self.sheet = self.workbook.active
-            self.lib = "openpyxl"
+            self.lib = LIB.OPENPYXL
         elif file.endswith(".xls"):
             self.workbook = xlrd.open_workbook(file, on_demand=True)
             self.sheet = self.workbook.sheet_by_index(0)
-            self.lib = "xlrd"
+            self.lib = LIB.XLRD
         else:
             raise ValueError("Unsupported file format")
 
+        if self.sheet is None:
```

Comment:
Consider moving this into the `if file.endswith(".xlsx")` block, since it's only really a possibility for .xlsx files.